archetype.rhai

```diff
@@ -23,9 +36,43 @@ context += set("organization-name", render("{{ org-name }}-{{ solution-name }}",
 });
 context += set("root-directory", render("{{ project-name }}", context));
 
+context += set("root-directory", render("{{ project-name }}", context));
+
+// context += prompt("Should we initialize a Git repository and stage the changes?:", "git_init", #{
+//     help: "Y/N",
+//     placeholder: "Y",
```

Comment:
you may want to add `defaults_with`, https://github.com/p6m-dev/java-spring-boot-grpc-service.archetype/blob/main/archetype.rhai#L32C5-L32C18
Useful when you want to run archtect in headless mode